package.json

```diff
@@ -21,16 +21,17 @@
 	"license": "MIT",
 	"author": {
 		"name": "Mario Beltrán Alarcón",
-		"email": "belco90@gmail.com",
+		"email": "me@mario.dev",
 		"url": "https://mario.dev/"
 	},
 	"main": "index.js",
 	"scripts": {
+		"prebuild": "del-cli dist",
```

Comment:
Deleting the previous `dist/` since it was causing issues for me when switching branches (the stale `dist/` wasn't cleared properly).